app/metrics.yaml

```diff
@@ -7328,26 +7315,6 @@ recent_bookmarks:
     expires: 107
     unit: integer
 
-recent_searches:
```

Comment:
The recently visited section is based on history, not tab groups, so this needs to stay if we want to retain groups in history.

Reply:
✅